Silicon/TigerlakePkg/Include/Register/SaRegsIpu.h

```diff
@@ -32,4 +32,7 @@
 // GPIO native features pins data
 //
 #define SA_GPIO_IMGUCLK_NUMBER_OF_PINS     2
+#ifdef PCH_ICL
+#define ICL_SA_GPIO_IMGUCLK_NUMBER_OF_PINS 5
+#endif

```

Comment:
we don't need add PCH_ICL back since it is not used for TGL.

Reply:
Done, I've removed all register with PCH_ICL.